hbase-hadoop2-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationSourceSourceImpl.java

```diff
@@ -163,10 +168,15 @@ public MetricsReplicationSourceSourceImpl(MetricsReplicationSourceImpl rms, Stri
     walEditsFilteredCounter.incr(size);
   }
 
-  @Override public void incrBatchesShipped(int batches) {
+  @Override
+  public void incrBatchesShipped(int batches) {
     shippedBatchesCounter.incr(batches);
   }
 
+  @Override public void incrFailedBatches() {
```

Comment:
ok i realize now that every other override in this class has the single-line. I am sure this will be changed in the rollout of spotless, but might just be fine to let yours be consistent with the rest of the class. sorry for the back and forth.